python/lsst/ts/standardscripts/auxtel/daytime_checkout/atpneumatics_checkout.py

```diff
@@ -124,22 +138,59 @@ async def run(self):
 
         await self.atcs.enable_ataos_corrections()
 
-        m1_pressure = await self.atcs.rem.atpneumatics.tel_m1AirPressure.aget(timeout=5)
-        self.log.info(
-            f"M1 Air pressure with enabled ATAOS corrections is {m1_pressure.pressure:0.0f} Pascals"
+        # Sleep to allow M1 to arrive at commanded pressure
```

Comment:
The way I was thinking this could be implemented is that you enable the correction, then make a loop that waits until the pressure arrives at the commanded pressure (+/- tolerance) and then you give it a delay. 

you may want to turn this into a method though:

```
await self.wait_pressure_buildup()

...

async def wait_pressure_buildup(self):

    time_start = time.time()
    while time.time() < time_start + self.timeout_pressure_buildup:
        (Check pressure, return when arrives in desired window)
    else:
        raise RuntimeError(f"M1 pressure failed to arrive at desired value ({m1_commanded_pressure}+/-{self.pressure_tolerance}) in specified time ({self.timeout_pressure_buildup}s)")
    
```

Reply:
I think the original version is a little simpler so will keep it, but I do like your version of the RuntimeError message.